doc/source/theory/FlavorSpace.rst

```diff
@@ -35,17 +35,18 @@ that e.g. in the proton will carry most of the momentum at large x and :math:`q^
 sea quark distribution:
 
 .. math ::
-    \mathcal F \sim \mathcal F_{\pm} = \span(g, u^+, u^-, d^+, d^-, s^+, s^-, c^+, c^-, b^+, b^-, t^+, t^-)
+    \mathcal F \sim \mathcal F_{\pm} = \span(\gamma, g, u^+, u^-, d^+, d^-, s^+, s^-, c^+, c^-, b^+, b^-, t^+, t^-)
 
 - this basis is *not* normalized with respect to the canonical Flavor Basis
 - the basis transformation to the Flavor Basis is implemented in
   :meth:`~eko.evolution_operator.flavors.rotate_pm_to_flavor`
 
-Evolution Basis
----------------
+QCD Evolution Basis
```

Comment:
maybe you can add an abbreviation in `abbreviation.rst` for QCD and QED and replace them everywhere: 
```
.. |QCD| replace::
   :abbr:`Quantum chromodynamics`

.. |QED| replace::
   :abbr:`... `

```